src/icons/_settings.scss

```diff
@@ -1,9 +1,6 @@
-@import "../settings";
-@import "../functions";
-
 // sizes
 $fd-icons-sizes: (
-  "default": $fd-font-size, // 14px
+  "default": 14px,
   "s": 12px,
   "m": 16px,
   "l": 20px,
```

Comment:
maybe these should be in rem?

Reply:
I am working on that and I will push soon

Reply:
deleted